### PR TITLE
Bring Back Admin Button

### PR DESCRIPTION
**Pull request by yattias**
- Bringing back admin button functionality w/new filtering mechanism

### CODE REVIEW

File: src/researchhub_document/related_models/document_filter_model.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from datetime import datetime, timedelta
 
 import pytz
+from django.contrib.postgres.fields import ArrayField
```
Comment (le-sun) [inline src/researchhub_document/related_models/document_filter_model.py:4]: Is this import being used?

File: src/researchhub_document/related_models/constants/document_type.py [hunk 2]
```diff
@@ -28,6 +28,9 @@
 FILTER_HAS_BOUNTY = "FILTER_HAS_BOUNTY"
 FILTER_OPEN_ACCESS = "FILTER_OPEN_ACCESS"
 FILTER_PEER_REVIEWED = "FILTER_PEER_REVIEWED"
+FILTER_HUBS_EXCLUDED = "FILTER_HUBS_EXCLUDED"
+FILTER_EXCLUDED_FROM_FEED = ("FILTER_EXCLUDED_FROM_FEED",)
+FILTER_INCLUDED_IN_FEED = ("FILTER_INCLUDED_IN_FEED",)
```
Comment (le-sun) [inline src/researchhub_document/related_models/constants/document_type.py:33]:
```suggestion
FILTER_EXCLUDED_FROM_FEED = "FILTER_EXCLUDED_FROM_FEED"
FILTER_INCLUDED_IN_FEED = "FILTER_INCLUDED_IN_FEED"
```

File: src/researchhub_document/related_models/constants/document_type.py [hunk 3]
```diff
@@ -28,6 +28,9 @@
 FILTER_HAS_BOUNTY = "FILTER_HAS_BOUNTY"
 FILTER_OPEN_ACCESS = "FILTER_OPEN_ACCESS"
 FILTER_PEER_REVIEWED = "FILTER_PEER_REVIEWED"
+FILTER_HUBS_EXCLUDED = "FILTER_HUBS_EXCLUDED"
```
Comment (le-sun) [inline src/researchhub_document/related_models/constants/document_type.py:31]: Is this needed?

**Review by le-sun — CHANGES_REQUESTED** [review]

File: src/researchhub_document/related_models/document_filter_model.py (hunk 1 above)
Comment (yattias) [reply src/researchhub_document/related_models/document_filter_model.py:4]: nope, good call!

File: src/researchhub_document/related_models/constants/document_type.py (hunk 2 above)
Comment (yattias) [reply src/researchhub_document/related_models/constants/document_type.py:33]: Right, you know what, I'm not sure why I did it this way. I almost swear linting formatted things like this lol

File: src/researchhub_document/related_models/constants/document_type.py (hunk 3 above)
Comment (yattias) [reply src/researchhub_document/related_models/constants/document_type.py:31]: Nope, it is an artifact. Good call!

**Pull request merged**